helios-benchmarks/build.gradle

```diff
@@ -58,3 +60,22 @@ task executeBenchmarks {
     }
```

Comment:
I'd store the benchmark generated by `executeHeliosBenchmark` in `build`

Reply:
solved here https://github.com/47deg/helios/pull/45